src/modules/keycloak-administration/domain/keycloak-user.service.ts

```diff
@@ -491,9 +494,10 @@ export class KeycloakUserService {
     }
 
     public async updateKeycloakUserStatus(
-        userId: string,
+        personId: string,
+        keyCloakUserId: string,
         enabled: boolean,
-        customAttributes?: Record<string, string>,
+        userLock: UserLock<true>,
```

Comment:
Feels weird to pass in both an `enabled` and the `userLock`. Shouldn't the existence of the lock tell you if the user should be locked/unlocked?

Reply:
Not exactly. The UserLock-Object is used here if a person is not locked to create a new lock in the database. If UserLock would not exist, it could not be created.